src/components/remote_control/src/resource_allocation_manager_impl.cc

```diff
@@ -82,13 +82,53 @@ AcquireResult::eType ResourceAllocationManagerImpl::AcquireResource(
                     "Current access_mode is AUTO_ALLOW. "
                         << "App: " << app_id << " is allowed to acquire "
                         << module_type);
+
       allocated_resources_[module_type] = app_id;
       return AcquireResult::ALLOWED;
     }
     default: { DCHECK_OR_RETURN(false, AcquireResult::IN_USE); }
   }
 }
 
+void ResourceAllocationManagerImpl::SetResourceState(
+    const std::string& module_type,
+    const uint32_t app_id,
+    const ResourceState::eType state) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  LOG4CXX_DEBUG(logger_,
+                "Setting state for " << module_type << " by app_id " << app_id
+                                     << " to state " << state);
+  const AllocatedResources::const_iterator allocated_it =
+      allocated_resources_.find(module_type);
+
+  DCHECK_OR_RETURN_VOID(allocated_resources_.end() != allocated_it)
+  LOG4CXX_DEBUG(logger_, "Resource " << module_type << " is acquired.");
```

Comment:
@dev-gh maybe to concatenate this log messages into one message, separated with '.' ?